lib/rubocop/cop/generator.rb

```diff
@@ -62,6 +62,10 @@ class %<cop_name>s < Base
                 # For example
                 MSG = 'Use `#good_method` instead of `#bad_method`.'
 
+                # optimization: don't call `on_send` unless the method name is in this list
```

Comment:
I think this is `TODO`.

```suggestion
                # TODO: Don't call `on_send` unless the method name is in this list
```